### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by nagasrihith608**
Corrected a couple of punctuations.

### CODE REVIEW

**Review by GamerBoomTV — CHANGES_REQUESTED** [review]
Hello @nagasrihith608,
thanks for helping us to make our Support++ software better. 

There must be no teaching characters between the commands (!online / !offline). 

Thanks a lot.

Kind Regards,
Dan

**Review by VerHext — APPROVED** [review]

**Review by Jniklas2 — APPROVED** [review]

**Review by esieben1310 — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -28,14 +28,14 @@ If you want to download the script, go to the `Releases Tab`_ and download the l
...
+* Module store - easily expands the script's functionality
 * Channel notifications
 * Ignore list
-* Messages (e.g. if no supporters are online)
+* Messages (e.g., if no supporters are online)
 * Create tickets
 * Send notifications via Telegram (module)
 * Send notifications via Discord (module)
-* Channel (name) renaming (!online / !offline)
+* Channel (name) renaming (! online /! offline)
```
Comment (Lapotor) [inline README.rst:38]: Please remove the space between `!` and online and offline

**Review by Lapotor — CHANGES_REQUESTED** [review]
Please change as commented.

**Comment by VerHext** [comment]
Unfortunately, no change has been made to this branch for over 10 days. I close this pull request and reject the changes. Feel free to open a new one anytime.

**Pull request closed without merging**